### DISCUSSION TITLE
Welcome.

### DISCUSSION THREAD

**Discussion by krishnemd** [org-45/readme#1] [Announcements]
Welcome.
This is the start of org45 discussion.
org45 name sounds something serious. It sounds like some entity that knows what it is doing and the task will be accomplished anyhow.

**Comment by Bibekdhkl** [comment]
So what's going to happen here?

**Comment by krishnemd** [comment]
@Bibekdhkl based on https://github.com/org-45/readme/issues/2 we will be having two teams at the moment. 

**Fullstack** team will collaborate to make a web based application. I think team of four will be fine to work on a single project.

For **devops** team we can start by creating POCs for deploying any of [these](https://github.com/awesome-selfhosted/awesome-selfhosted) open source alternatives locally.

You can also get idea on what we are doing via this https://github.com/orgs/org-45/projects section.
